experiment-setup/execution/host_sequence/scenario_api.py

```diff
@@ -28,12 +27,13 @@ def request_endpoint(path, method="get", data=None, params=None, user_session=No
 
 def sequential_interval_scenario(service, start, iter):
     # set main user
-    main_user = api_register_data[0]["username"]
+    main_user = api_register_data_dummie(0)["username"]
 
     # 1. Register all users
     print_info_call("API", service, "Register", ITER_NUM)
     api_register_response = []
```

Comment:
Append the reposes to the results `results.append(api_register_response) and return the results in the end with `return results`, leaving out:
```
[f"{service}-api-{start}-{iter}",
            api_register_response,
            api_message_response,
            api_public_msgs,
            api_user_msgs,
            api_follow_user_1,
            api_user_followers,
            api_unfollow_user_1,
            api_latest]
```